JS/models/militaryType.js

```diff
@@ -1,7 +1,7 @@
 const MilitaryType = {
```

Comment:
The same as for comment regarding Experimantal Types.

Reply:
All values changed due to CAPITAL_LETTERS notation